src/app/angularjs/tabs.component.ts

```diff
@@ -18,11 +18,15 @@ export const tabsComponent = {
         }
       });
     }
+
+    showSimpleToast(message: string): void {
+      this.$mdToast.show(this.$mdToast.simple().textContent(message).hideDelay(2000));
+    }
   },
 };
 
 @Directive({ selector: tabsComponent.selector })
-export class TabsComponentFacade extends UpgradeComponent {
+export class TabsComponentDirective extends UpgradeComponent {
```

Comment:
We have typically called these facades in `ngUpgrade` terminology, because they are just facades for the AngularJS component (they don't have any other logic or purpose).

I don't feel strongly about it, but I do like "Facade", which distinguishes these from regular, general-purpose directives.

Reply:
💯 But then you will get lint errors, as the `angular-eslint/recommended` plugin expects "Directive" word as a suffix. We can override the rule. Let us know if we should

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/5073455/223485964-4e3b842d-f196-4daf-a65a-c570a10b7c9b.png">

Reply:
I see. I don't feel strongly about it, but what I would personally do in hybrid apps (such as this one) is configure the [directive-class-suffix][1] rule to accept both `Directive` and `Facade`. But just using `Directive` is also fine by me. (There's also the option of using `...FacadeDirective` 🤔).

Leaving it up to you - I am OK either way :smiley:

[1]: https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin/docs/rules/directive-class-suffix.md

Reply:
Changed eslint rules to accept both: `Directive` and `Facade`.
Also changed class names from `TabsComponentDirerctive` to `TabsComponentFacade` and from `VersionStampComponentDirective` to `VersionStampComponentFacade`